lib/scope/scope.go

```diff
@@ -0,0 +1,152 @@
+package scope
+
+import (
+	"github.com/analog-substance/arsenic/lib/util"
+	"github.com/spf13/viper"
+	"golang.org/x/net/publicsuffix"
+	"strings"
+)
+
+type scope struct {
+	domainsExplicitlyInScope []string
+	rootDomainsExplicitlyInScope []string
+	blacklistedRootDomains []string
+	blacklistedDomains []string
+	explicitDomainsLoaded bool
+
+	hostIPsExplicitlyInScope []string
+	hostIPsExplicitlyInScopeLoaded bool
+}
+
+var asScope = &scope{}
+var asScopeInit = false
```

Comment:
This just for the test? go-staticcheck tells me that it isn't being used anywhere

Reply:
i think this can be deleted. It was remnants from earlier code